frame/composable-support/src/rpc_helpers.rs

```diff
@@ -1,79 +1,42 @@
-use codec::{Codec, Decode, Encode};
+use codec::{Decode, Encode, MaxEncodedLen};
 #[cfg(feature = "std")]
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
-
-/// https://github.com/interlay/interbtc/blob/a7c0e69ac041176a2531bafb1c4e35cbc2f7e192/crates/oracle/rpc/runtime-api/src/lib.rs#L10
+#[derive(
+	scale_info::TypeInfo,
+	Clone,
+	Copy,
+	Debug,
+	Decode,
+	Default,
+	Encode,
+	Eq,
+	MaxEncodedLen,
+	Ord,
+	PartialEq,
+	PartialOrd,
+)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-#[derive(Debug, PartialEq, Eq, Encode, Decode)]
-pub struct SafeRpcWrapper<T: SafeRpcWrapperType>(
-	#[cfg_attr(feature = "std", serde(serialize_with = "serialize_to_hex"))]
```

Comment:
What was the reason to use hex representation before in the first place?
I guess I need to change [my PR](https://github.com/ComposableFi/composable/pull/900) based on this as well

Reply:
I'm not sure tbh.